js/src/common/Application.js

```diff
@@ -297,6 +305,10 @@ export default class Application {
       options.method = 'POST';
     }
 
+    //Save a reference to the request object, that can be used to abort the request when the user changes routes while the request is still loading.
```

Comment:
Extremely minor nitpick, but can we add a space before the start of comment text and split the comment over 2 lines?

Also for the conditional below, can we wrap that in braces?